src/Faker/Provider/ar_PS/Lorem.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
lorem in arabic here:
https://ar.wikipedia.org/wiki/%D9%84%D9%88%D8%B1%D9%8A%D9%85_%D8%A5%D9%8A%D8%A8%D8%B3%D9%88%D9%85